python/afdko/ufotools.py

```diff
@@ -2288,58 +2288,37 @@ def checkHashMaps(fontPath, doSync):
     msgList = []
     allMatch = True
 
-    useHashMap = True
-    programName = "CheckOutlines"
-    ufoFontData = UFOFontData(fontPath, useHashMap, programName)
-    gm = ufoFontData.getGlyphMap()
-    gNameList = gm.keys()
+    ufoFontData = UFOFontData(fontPath, True, 'programName')
```

Comment:
That string is required by the method, but in this use case its value is irrelevant because what gets compared downstream is just the hash portion — the list of program names is ignored.

Reply:
👍

Reply:
we can't because that method is called from other places where that string is important

Reply:
OK... maybe add a comment that `'programName'` isn't used in this part of the code? Perhaps called it `'dummyValue'` or just `''`. I'm just thinking that as it is, it confuses me now and will confuse me (or whoever edits this code next) more later.  😄

Reply:
I considered using `'placeholder'` but then ended simply quoting what was there.
I'll change it to `'this value intentionally left blank'` 😆

Reply:
APPROVED 😆